sunpy/database/database.py

```diff
@@ -381,6 +381,7 @@ def _download_and_collect_entries(self, query_result, **kwargs):
         path = kwargs.pop('path', None)
         progress = kwargs.pop('progress', False)
         methods = kwargs.pop('methods', ('URL-FILE_Rice', 'URL-FILE'))
+        caching = kwargs.pop('caching', True)
```

Comment:
We discussed changing this keyword, I think I am personally in favour of `'overwrite'` being the opposite of `'caching'` i.e. defaults to `False`.